lms/djangoapps/philu_overrides/branding/api.py

```diff
@@ -43,7 +43,7 @@ def get_auth_footer(is_secure=True):
     }
 
 
-def get_non_auth_footer(is_secure=True):
+def get_non_auth_footer():
```

Comment:
Since we have removed the argument, lets also create a PR for theme side, to remove the argument from caller end as well.

Reply:
Surely, Already on it